### PR TITLE
[MDEP-966] Add previously undeclared but used sisu dependency

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

**Review by Bukama — APPROVED** [review]
I had in mind that there is an enforce rule that ensures to all dependencies must be defined explicitly, but I don't find it (or I remember wrong). But so from my amateurish (in this context) point of view this explicit declaration is the way we should do it.

so non binding approve by me.

File: pom.xml [hunk 1]
```diff
@@ -171,6 +171,11 @@ under the License.
       <artifactId>plexus-archiver</artifactId>
       <version>${plexus-archiver.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.sisu</groupId>
+      <artifactId>org.eclipse.sisu.plexus</artifactId>
+      <version>0.9.0.M3</version>
+    </dependency>
```
Comment (slawekjaranowski) [inline pom.xml:178]: please check if it is provided by Maven

Comment (elharo) [reply pom.xml]: done

**Review by slachiewicz — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #1480

**Pull request merged**